### ISSUE TITLE
How does the SpringBook project read the local configuration file to start when Nacos is not online

### ISSUE THREAD

**Issue by gaofeicm** [alibaba/nacos#1024]
1. nacos包下载来，未做其他配置，启动运行，在配置管理添加了自己的配置，Springboot项目使用Nacos作为配置中心

- The Nacos package was downloaded, no other configuration was done, and it started running. It added its own configuration in configuration management. The Springboot project uses Nacos as the configuration center.

2. 在项目的`bootstrap.properties`配置文件中加入`spring.cloud.nacos.config.group`和`spring.cloud.nacos.config.server-addr`，当Naocs在线的时候，项目正常启动，当Nacos不在线的时候，项目无法启动，控制台报如下错误：

- Add `spring.cloud.nacos.config.group` and `spring.cloud.nacos.config.server-addr` to the `bootstrap.properties` configuration file of the project. When Naocs is online, the project starts normally. When Nacos is not online, the project cannot start. The console reports the following error：

- ![image](https://user-images.githubusercontent.com/25575562/55775316-54424180-5acb-11e9-9a9b-fa9777c4f75a.png)

3.    @yanlinly 这个问题该如何解决？
- @yanlinly How can this problem be solved?

**Comment by nkorange** [comment]
@gaofeicm This is a client problem. What's your client version?

**Comment by gaofeicm** [comment]
> @gaofeicm This is a client problem. What's your client version?
nacos-client-1.0.0.jar
![image](https://user-images.githubusercontent.com/25575562/56786909-3562df80-682d-11e9-8ebc-840f7023dc33.png)
![image](https://user-images.githubusercontent.com/25575562/56786913-3bf15700-682d-11e9-914e-7ed91ab7f773.png)

**Comment by nkorange** [comment]
If you are using Spring Boot, follow the [example](https://github.com/nacos-group/nacos-spring-boot-project/blob/master/nacos-spring-boot-samples/nacos-config-sample/src/main/resources/application.properties) to configure your server address.

**Comment by gaofeicm** [comment]
> If you are using Spring Boot, follow the [example](https://github.com/nacos-group/nacos-spring-boot-project/blob/master/nacos-spring-boot-samples/nacos-config-sample/src/main/resources/application.properties) to configure your server address.

I have the same configuration. The difference is that my service address is not localhost. Now the problem is that if I can't connect to the Nacos server, the project can't start. I consulted before that configuration will cache the local one. If I can't connect to the server, project will use the configuration of the local cache. I checked my local cache, it does exist, but when the project starts, it will report an error.
